sdk/go/auto/errors_test.go

```diff
@@ -179,8 +179,7 @@ func TestCompilationErrorGo(t *testing.T) {
 	}()
 
 	_, err = s.Up(ctx)
-	assert.Error(t, err)
```

Comment:
There were quite a few tests we we asserted we had an error than asserted something about that error. We can just is the second assert in all these cases as it will fail if `err==nil` which is all the first assert was checking.